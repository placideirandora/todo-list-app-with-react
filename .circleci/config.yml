version: 2.1
jobs:
  build:
    docker:
      - image: circleci/node:10.16.0

    working_directory: ~/todo-list-app-with-react-js

    steps:
      - checkout
      - restore_cache:
          keys:
            - v1-dependencies-{{ checksum "package.json" }}
            - v1-dependencies-

      - run: npm install

      - save_cache:
          paths:
            - node_modules
          key: v1-dependencies-{{ checksum "package.json" }}

      - run: npm run test:coverage
