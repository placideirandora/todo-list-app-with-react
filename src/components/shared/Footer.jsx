import React, { Fragment } from 'react';
import '../../styles/Footer.scss';

const Footer = () => {
  return (
    <Fragment>
      <p>
        Copyright &copy; 2019 - {new Date().getFullYear()} -- TO-DO LIST -
        Placide IRANDORA
      </p>
    </Fragment>
  );
};

export default Footer;
