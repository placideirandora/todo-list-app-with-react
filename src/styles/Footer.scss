@import 'Colors';
@import 'Fonts';

p {
  text-align: center;
  margin-top: 20rem;
  font-size: 25px;
  font-family: $saira;
}
