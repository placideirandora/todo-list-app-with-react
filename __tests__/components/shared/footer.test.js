import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as adapter from '../../config/enzyme.config';
import Footer from '../../../src/components/shared/Footer';

describe('Footer Component Test Cases', () => {
  it('should render the component without any failure', () => {
    const component = shallow(<Footer />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
