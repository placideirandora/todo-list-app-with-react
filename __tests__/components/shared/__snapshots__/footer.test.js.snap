// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Test Cases should render the component without any failure 1`] = `
<Fragment>
  <p
    style={
      Object {
        "color": "grey",
        "fontFamily": "Saira Stencil One",
        "fontSize": "25px",
        "textAlign": "center",
      }
    }
  >
    Copyright Â© 2019 - 
    2020
     -- TO-DO LIST - Placide IRANDORA
  </p>
</Fragment>
`;
